src/main/scala/io/iohk/ethereum/network/AuthInitiateMessage.scala

```diff
@@ -5,16 +5,16 @@ import io.iohk.ethereum.crypto._
 import org.spongycastle.math.ec.ECPoint
 
 object AuthInitiateMessage {
-  val SignatureSize = 65
+  val encodedLength = ECDSASignature.encodedLength + 32 + 64 + 32 + 1
 
   def decode(input: Array[Byte]): AuthInitiateMessage = {
-    val publicKeyIndex = SignatureSize + 32
+    val publicKeyIndex = ECDSASignature.encodedLength + 32
```

Comment:
same, here, it would be nicer to read if I know that 32 is for example package length or hash lengt